docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -6,19 +6,22 @@
 
 preview::[]
 
+Updates the draft `filtering` configuration of a connector and marks the draft validation state as `edited`. The draft filtering can be then activated after a successful validation of the draft by the {enterprise-search-ref}/connectors.html[Elastic connector service].
```

Comment:
I wonder if we should make it explicit earlier that filtering is synonymous with sync rules

Reply:
++

yeah, maybe a paragraph below we should add extra paragraph that `rules` == `basic sync rules` , `advanced_snippet` == `advanced sync rules`?